src/aind_metadata_service/sharepoint/client.py

```diff
@@ -44,6 +47,11 @@ class ProcedureType(Enum):
         INJECTION = "Injection"
         INJ = "INJ"
         OPTIC_FIBER_IMPLANT = "Optic Fiber Implant"
+        HP_TRANSCRANIAL = "HP Transcranial (for ISI)"
+        WHOLE_HEMISPHERE_CRANIOTOMY_NP = "WHC NP"
```

Comment:
shouldn't the spelled out part be the other side - the part in quotes? The acronym from sharepoint on the left and what it means on the right? Or do I not understand how you are using this?

Reply:
the right part in quotes is how its labeled in sharepoint